prql-compiler/src/sql/mod.rs

```diff
@@ -30,13 +75,40 @@ pub struct Options {
     /// If None is used, `sql_dialect` flag from query definition is used.
     /// If it does not exist, [Dialect::Generic] is used.
     pub dialect: Option<Dialect>,
+
+    /// Emits the compiler signature as a comment after generated SQL
+    ///
+    /// Defaults to true.
+    pub signature_comment: bool,
 }
 
 impl Default for Options {
     fn default() -> Self {
         Self {
             format: true,
             dialect: None,
+            signature_comment: true,
         }
     }
 }
+
+impl Options {
+    pub fn no_format(mut self) -> Self {
```

Comment:
I had a look at how bools were generally handled in the builder pattern; one factory crate just uses the field name for them all: https://docs.rs/derive_builder/latest/derive_builder/

It at least makes sense to me to handle bools similarly to others

(this is nitty though, no need to change)

Reply:
This is a small digression from the pattern and I would argue it makes more sense here.